Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on themes/src/main/resources-community/theme/base/account/messages/messages_fr.properties:
```diff
@@ -261,3 +261,59 @@ authorizedProvider=Fournisseur autorisé
+error-user-attribute-required=Merci de renseigner l''attribut {0}.
+error-invalid-date=Date non valide.
+error-user-attribute-read-only=Le champ {0} est en lecture seule.
+error-username-invalid-character=Le nom d'utilisateur contient un caractère non valide.
```
@GitSpoon - please note that this translation contains only one single quote where it should contain two single quotes. See [the translation docs](https://github.com/keycloak/keycloak/blob/main/docs/translation.md#handling-of-single-quotes-) for more information. The automatic check in flagged those messages, and I'm now updating weblate. 

> Whenever messages are formatted in the backend, Keycloak uses Java's MessageFormat to replace placeholders. This uses
single quotes (`'`) as an escape mechanism.
>
> Therefore, use typographic quotes like `‘` and `’` where possible.
> 
> To avoid problems, automatic checks ensure the following:
> 
> * In all "base" themes, where message keys are evaluated in the backend, no single quote (`'`) must be used standalone as it would not print when used in Java's MessageFormat.
>  For legacy messages, a double single quote (`''`) can be used to print a single quote.
> 
> * In all UI themes, where message keys are evaluated in the frontend, a single quote (`'`) must only be used standalone, and a double single (`''`) must not be used.

weblate committed:
Updated translation for German

Language: de

Co-authored-by: Johannes Bergmann <johbergmann@web.de>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Johannes Bergmann <johbergmann@web.de>

weblate committed:
Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Co-authored-by: Takashi Norimatsu <takashi.norimatsu.ws@hitachi.com>
Co-authored-by: Yoshiyuki Tabata <yoshiyuki.tabata.jy@hitachi.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Signed-off-by: Takashi Norimatsu <takashi.norimatsu.ws@hitachi.com>
Signed-off-by: Yoshiyuki Tabata <yoshiyuki.tabata.jy@hitachi.com>

weblate committed:
Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Updated translation for French

Language: fr

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Dodouce <marin.pau22@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Sylvain Pichon <service@spichon.fr>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Dodouce <marin.pau22@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Sylvain Pichon <service@spichon.fr>

ahus1 commented:
@y-tabata / @wadahiro / @tnomrimat: The Japanese changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@jmorille / @blackholegalaxy: The French changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.
